examples/somersaultecu.py

```diff
@@ -1780,7 +1781,7 @@ class SomersaultSID(IntEnum):
                 ProgCode(
                     code_file="jobs.jar",
                     encryption=None,
-                    syntax="JAR",
+                    syntax=ProgCodeSyntax.JAR,
```

Comment:
Unrelated to PR

Reply:
true. It's IMO too small to warrant its own PR, so I at least moved this into a separate patch...